src/WebJobs.Script.WebHost/Controllers/HostController.cs

```diff
@@ -107,6 +110,56 @@ public async Task<IActionResult> GetHostStatus([FromServices] IScriptHostManager
             return Ok(status);
         }
 
+        /// <summary>
+        ///  Regarding security, GET will allow Readers to list this info, which is OK.
```

Comment:
```suggestion
        ///  Currently, all Reader roles can access this information. 
        /// If this API is extended to include any secrets, it will need to be 
        /// locked down to only Contributor roles.  
```